.travis.yml

```diff
@@ -35,11 +35,9 @@ install:
  - true
 
 script:
-  - ./gradlew build idea -s -PcheckJava6Compatibility
+  - ./gradlew build idea -s -PcheckJava6Compatibility && ./gradlew ciPerformRelease
```

Comment:
>Cant this just be ./gradlew build idea -s -PcheckJava6Compatibility ciPerformRelease ?

It's not that safe because Gradle does various optimizations to make the build faster. So we can be still running tests while some other Gradle task will be shipping libraries to Bintray :)